api/migrations/versions/1349a2c924f4_.py

```diff
@@ -0,0 +1,281 @@
+                ],
+                'is_required': False,
+                'order': 8,
+                'weight': 0
+            },
+            {
+                'id': 9,
+                'review_form_id': 1,
+                'question_id': 8,
+                'headline': None,
+                'description': None,
+                'type': 'information',
+                'options': None,
+                'is_required': False,
+                'order': 9,
+                'weight': 0
+            },
+            {
+                'id': 10,
+                'review_form_id': 1,
+                'question_id': 11,
+                'headline': None,
+                'description': None,
+                'type': 'information',
+                'options': None,
+                'is_required': False,
+                'order': 10,
+                'weight': 0
+            },
+            {
+                'id': 11,
+                'review_form_id': 1,
+                'question_id': 9,
+                'headline': None,
+                'description': None,
+                'type': 'multi-choice',
+                'options': [
+                    {'label': 'Poor', 'value': 0},
+                    {'label': 'Below Average', 'value': 1},
+                    {'label': 'Average', 'value': 2},
+                    {'label': 'Above Average', 'value': 3},
+                    {'label': 'Excellent', 'value': 4}
+                ],
+                'is_required': False,
+                'order': 11,
+                'weight': 0
+            },
+            {
+                'id': 12,
+                'review_form_id': 1,
+                'question_id': 17,
+                'headline': None,
+                'description': None,
+                'type': 'information',
+                'options': None,
+                'is_required': False,
+                'order': 12,
+                'weight': 0
+            },
+            {
+                'id': 13,
+                'review_form_id': 1,
+                'question_id': 18,
+                'headline': None,
+                'description': None,
+                'type': 'information',
+                'options': None,
+                'is_required': False,
+                'order': 13,
+                'weight': 0
+            },
+            {
+                'id': 14,
+                'review_form_id': 1,
+                'question_id': None,
+                'headline': 'Final Verdict',
+                'description': 'Do you think this applicant should attend the Indaba?',
+                'type': 'multi-choice',
+                'options': [
+                    {'label': 'No', 'value': 0},
+                    {'label': 'Maybe', 'value': 1},
+                    {'label': 'Yes', 'value': 2}
+                ],
+                'is_required': True,
+                'order': 14,
+                'weight': 0
+            }
+        ]
+    )
+
+    # ### end Alembic commands ###
```

Comment:
Since you're using bulk_insert, please (re)set the ID sequence on these tables to prevent issues with future inserts:

op.execute("""SELECT setval('review_question_id_seq', (SELECT max(id) FROM review_question));""")
op.execute("""SELECT setval('review_form_id_seq', (SELECT max(id) FROM review_form));""")

(also do this in the downgrade)

Reply:
Cool done.